### ISSUE TITLE
Overflow menu examples missing aria-expanded

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#3773]
The examples are missing `aria-expanded`. It looks like there is some logic for this in https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/OverflowMenu/overflow-menu-control.hbs referencing `overflow-menu-dropdown-button-aria-expanded` but doesn't appear to be working.
